sentry-android-core/src/main/java/io/sentry/android/core/AppComponentsBreadcrumbsIntegration.java

```diff
@@ -85,41 +85,20 @@ public void close() throws IOException {
   @SuppressWarnings("deprecation")
   @Override
   public void onConfigurationChanged(@NotNull Configuration newConfig) {
-    if (hub != null) {
-      final Device.DeviceOrientation deviceOrientation =
-          DeviceOrientations.getOrientation(context.getResources().getConfiguration().orientation);
-
-      String orientation;
-      if (deviceOrientation != null) {
-        orientation = deviceOrientation.name().toLowerCase(Locale.ROOT);
-      } else {
-        orientation = "undefined";
-      }
-
-      final Breadcrumb breadcrumb = new Breadcrumb();
-      breadcrumb.setType("navigation");
-      breadcrumb.setCategory("device.orientation");
-      breadcrumb.setData("position", orientation);
-      breadcrumb.setLevel(SentryLevel.INFO);
-
-      final Hint hint = new Hint();
-      hint.set(ANDROID_CONFIGURATION, newConfig);
-
-      hub.addBreadcrumb(breadcrumb, hint);
-    }
+    executeInBackground(() -> captureConfigurationChangedBreadcrumb(newConfig));
```

Comment:
i'm a little bit worried how this would work with session replay, because we're listening to orientation changes and change the replay orientation, but if the queue is busy here, the breadcrumb will be out-of-sync. Can we maybe keep this breadcrumb on the main thread still?

Reply:
Let me switch to the new breadcrumb ctor with timestamp instead, then it should be fine.